target/linux/mpc85xx/base-files/lib/upgrade/platform.sh

```diff
@@ -14,7 +14,8 @@ platform_do_upgrade() {
 
 	case "$board" in
 	ocedo,panda|\
-	sophos,red-15w-rev1)
+	sophos,red-15w-rev1|\
+	edgecore,ecw7210)
```

Comment:
Please take care of alphabetic sorting. Despite, the device should be named edgecore,ecw7210-l (or edge-core,ecw7210-l)